extensions/whatsapp/CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Whatsapp Changelog
+
+## [Empty Screen] - 2022-03-26
+Add empty screen for the `Open Chat` command to avoid confusions
+
+## [Categories & Screenshots] - 2022-03-12
+Add categories and screenshots for the store
+
+## [Added Whatsapp] - 2021-12-06
+Initial version code
```

Comment:
@mattisssa you did something once with a README of my WhatsApp extension. Now it appears as a change each time I develop haha

Could we please fix it? I've tried everything but it always appears as a change.

Reply:
Here is the thing you did I mentioned: 
https://github.com/raycast/extensions/pull/1087/commits/396ecd6da375703b3ea1e1d3265eb0e27ff88a4b

Reply:
Hey! Sorry, I didn't want to make an issue from my change 😬 
Have you rebased the change I made previously? All I did is just renamed the lowercased changelog file into uppercased. If you did the rebase I guess there still can be your old `changelog.md` file that can be removed now 🤔

Reply:
Do you need anything from us @vimtor

Reply:
I am not a 100% sure how to fix this. If you don't have a problem with this file being here I don't have more to say 😄

Reply:
@vimtor let's leave it there then 😬